pennylane/measurements/mid_measure.py

```diff
@@ -173,6 +173,13 @@ def func(x):
             With analytic execution, ``qml.mutual_info`` will raise errors when using any interfaces except
             ``jax``, and ``qml.vn_entropy`` will raise an error with the ``tensorflow`` interface when the
             postselection state has zero probability.
+
+        .. warning::
```

Comment:
this seems related to the above warning - can they be re-organized and combined into a single warning? basically 3 different cases of when things break while post-selecting on a zero-probability state